integration-tests/test-extension/extension/deployment/src/main/resources/application.properties

```diff
@@ -152,8 +152,8 @@ quarkus.arc.unremovable-types=foo
 # The YAML source may add an indexed property (depending on how the YAML is laid out). This is not supported by @ConfigRoot
 quarkus.arc.unremovable-types[0]=foo
 
-### Do not record env values in build time
-bt.ok.to.record=properties
+### recording
+bt.ok.to.record=from-app
 %test.bt.profile.record=properties
```

Comment:
Hi @radcortez, quick question, does `%test.bt.profile.record=properties` mean that it is OK to record the `bt` profile config property values in the test mode but the recording for this profile is off in the prod mode ?

Reply:
We record all profiled names. We need to ensure that the recorded properties retain their original property name, to not override each other. Local sources are excluded from any recording for any case with this PR.